requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+flake8
+pytest
+suds-jurko
+requests
```

Comment:
I was [actually agreeing with you](65#discussion_r111423945) on removing the two dependencies ...